docs/source/conf.py

```diff
@@ -144,7 +145,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+html_static_path = []
```

Comment:
I don't understand this change, could you explain?

Reply:
Sphinx gives a warning that this path doesn't exist. It's not needed.  In Cartographer this was already fixed: https://github.com/googlecartographer/cartographer/pull/1055